src/service/template-impl.js

```diff
@@ -285,7 +288,9 @@ export class Templates {
     } else if (opt_querySelector) {
       return scopedQuerySelector(parent, opt_querySelector);
     } else {
-      return childElementByTag(parent, 'template');
+      const template = childElementByTag(parent, 'template');
+      const templateByScript = childElementByTag(parent, 'script');
```

Comment:
Use a query selector to do this in one pass.

Reply:
done.